src/wallet/rpcwallet.cpp

```diff
@@ -5522,7 +5527,12 @@ UniValue z_shieldcoinbase(const UniValue& params, bool fHelp)
                 }
             },
             [&](const libzcash::UnifiedAddress& ua) {
-                // OK
+                if (!ua.GetSaplingReceiver().has_value()) {
+                    throw JSONRPCError(
+                            RPC_VERIFY_REJECTED,
+                            "Only Sapling shielding is currently supported by z_shieldcoinbase. "
+                            "Use z_sendmany with a transaction amount that results in no change for Orchard shielding.");
```

Comment:
Is there an issue to fix this?

Reply:
Yes, https://github.com/zcash/zcash/issues/5511; I'm presently working on it.